.github/workflows/build.yml

```diff
@@ -0,0 +1,14 @@
+on: [push, fork, pull_request]
+jobs:
+  lint-test-build-package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.50.1
+      - run: go version
```

Comment:
```suggestion
```

Sorry for going back and forth, but I am also leaning:

![image](https://user-images.githubusercontent.com/1142578/205971176-e87dd602-708d-4a49-bb47-8f0320ab616f.png)

Reply:
np